testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -72,6 +72,24 @@
   node.run("#{salt_call} saltutil.refresh_grains")
 end
 
+When(/^I setup a git_pillar environment on the Salt Master$/) do
+  file = 'salt_git_pillar_setup.sh'
+  source = "#{File.dirname(__FILE__)}/../upload_files/#{file}"
+  dest = "/tmp/#{file}"
+  success = file_inject(get_target('server'), source, dest)
```

Comment:
All cool, but jfyi, we also support this:
```
get_target('server').inject(source, dest)
```

Reply:
yup, would be better. I missed that one.

Reply:
Is this just costmetic or really a requirement for PR approval? I see the form `success = file_inject(get_target('server'), source, dest)` is used also in other parts of the code.

Reply:
Cosmetic. No worries.